client/scss/components/forms/_input-base.scss

```diff
@@ -0,0 +1,32 @@
+/**
+ * Field styles reusable across **all** fields, including:
+ * Text input, textarea, checkbox, radio, select, etc.
+ */
+@mixin input-base() {
+  appearance: none;
+  border-radius: theme('borderRadius.DEFAULT');
+  color: $color-input-text;
+  background-color: $color-input-bg;
+  border: 1px solid $color-input-border;
+
+  &:hover {
+    border-color: $color-input-hover-border;
+  }
+
+  &[disabled],
+  &[disabled]:hover {
+    color: $color-input-disabled-text;
+    background-color: $color-input-disabled-bg;
+    border-color: $color-input-disabled-border;
+    cursor: not-allowed;
+  }
+
+  .error &,
+  &[aria-invalid='true'] {
+    border-color: $color-input-error-border;
```

Comment:
Should we also use invalid pseudo selector while we are at it?

`&:invalid, `

https://developer.mozilla.org/en-US/docs/Web/CSS/:invalid

Reply:
My understanding is that this would only be relevant if we used browser validation, which we’ve specifically disabled. Since our errors are server-rendered, the fields technically wouldn’t be `:invalid`. I’ll leave this un-resolved though in case I got it wrong.

Reply:
Good call. However - there may be other fields that add this anyway - clientside. 

Happy to suggest we do that in a future improvement though.

Reply:
👍 good point, I hadn’t considered this. I’ll add this to the list of refactorings to consider later.